README.rst

```diff
@@ -1,4 +1,123 @@
-Microsoft Azure SDK for Python
-==============================
+Microsoft Azure Storage SDK for Python
+======================================
 
-This is the Microsoft Azure Storage Runtime Client Library.
+This project provides a client library in Python that makes it easy to
+consume Microsoft Azure Storage services. For documentation please see
+the Microsoft Azure `Python Developer Center`_ and our `ReadTheDocs`_ page.
+
+    If you are looking for the Service Bus or Azure Management
+    libraries, please visit
+    https://github.com/Azure/azure-sdk-for-python.
+
+Features
+========
+
+-  Blob
+
+   -  Create/Read/Update/Delete containers
+   -  Create/Read/Update/Delete blobs
+   -  Advanced Blob Operations
+
+-  Queue
+
+   -  Create/Delete Queues
+   -  Insert/Peek Queue Messages
+   -  Advanced Queue Operations
+
+-  Table
+
+   -  Create/Read/Update/Delete tables
+   -  Create/Read/Update/Delete entities
+   -  Batch operations
+   -  Advanced Table Operations
+
+-  Files
+
+   -  Create/Update/Delete Directories
+   -  Create/Read/Update/Delete Files
+   -  Advanced File Operations
+
+Getting Started
+===============
+
+Download
+--------
+
+Option 1: Via PyPi
+~~~~~~~~~~~~~~~~~~
+
+To install via the Python Package Index (PyPI), type:
+::
+
+    pip.exe install azure-storage
```

Comment:
Also, we'll need to add a section that advises them to uninstall the old `azure` package before installing the new `azure-storage`.

This is my draft, not really happy with it yet:
https://github.com/Azure/azure-sdk-for-python/tree/master/azure

If you have suggestions, let me know.  It all comes down to not being able to have both the old azure<1.0 and the new azure-storage at the same time.  Installing azure-storage won't uninstall the old azure.  The only way to uninstall the old azure is to:
- pip uninstall azure
- pip install azure --upgrade
ie. a package can't uninstall another package as part of its installation.